src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -330,8 +334,9 @@ export default class rd2EntryForm extends LightningElement {
     */
     evaluateElevateEditWidget(paymentMethod) {
         let statusField = getFieldValue(this.record, FIELD_STATUS);
+        this.commitmentId = getFieldValue(this.record, FIELD_COMMITMENT_ID);
 
-        if(this.isEdit && statusField != STATUS_CLOSED){
+        if(this.isEditEnabled && this.commitmentId !== null && this.isEdit && statusField != STATUS_CLOSED){
```

Comment:
```suggestion
        if (this.isEditEnabled && this.commitmentId !== null && this.isEdit && statusField !== STATUS_CLOSED){
```

Reply:
I definitely need to look at the style guide again, thanks for pointing this out.